src/components/Toast/Toast.styled.ts

```diff
@@ -47,38 +48,87 @@ export const SCounter = styled.div`
 `
 
 export const SClose = styled(Close)`
+  min-width: 24px;
+  min-height: 24px;
+
+  color: ${theme.colors.white};
+  background: #0e132f;
+  transition: background ${theme.transitions.default};
+
   position: absolute;
-  top: -4px;
-  right: -4px;
+  top: -12px;
+  right: -8px;
   overflow: hidden;
 
+  border: 1px solid #30344c;
+
   display: flex;
   align-items: center;
   justify-content: center;
 
-  width: 24px;
-  height: 24px;
+  cursor: pointer;
 
-  color: ${theme.colors.neutralGray100};
-  background-color: ${theme.colors.backgroundGray800};
-  border: none;
-  border-radius: 9999px;
+  padding: 0;
 
-  padding: 4px;
+  svg {
+    width: 14px;
+    height: 14px;
+  }
 
   &:hover {
-    cursor: pointer;
+    background: ${theme.colors.basic500};
+    border: none;
   }
 `
 
+export const SProgressBarBg = styled.div`
+  background: ${theme.colors.basic700};
+
+  width: 100%;
+  height: 1.6px;
```

Comment:
nepouzivat desetinne hodnoty - bud 1px nebo 2px

Reply:
co kdyz toto je v designu?
<img width="635" alt="Screenshot 2022-11-10 at 11 40 39" src="https://user-images.githubusercontent.com/115090120/201069925-47cace16-ba20-42b1-97dd-06c74bec2c33.png">

Reply:
tak to zaokrouhlime my, browsery to stejne budou resit po svojem a bude to pak nekonzistentni

Reply:
okej, dal jsem to na 1px